### DISCUSSION TITLE
Enable GitHub Discussions & Move all Suggestions & Questions to that!

### DISCUSSION THREAD

**Discussion by TomLewis** [Zrips/CMI#4728] [Feature Requests]
GitHub added a forum type system with categories called Discussions that you can enable for each repo.
You could keep issues as actual issues/bugs and move all Questions/Suggestions into the Discussions tab to keep things much tidier.

https://github.blog/2020-05-06-new-from-satellite-2020-github-codespaces-github-discussions-securing-code-in-private-repositories-and-more/

**Comment by Zrips** [comment]
Yea, really good idea as i wanted to move suggestions away from ticket category already. This will be perfect. Tho there isint a way to mass export all tickets with feature request over there. So it might be some what tedious job to do it one by one.

**Comment by TomLewis** [comment]
You can suggest features, having multiple selection tools would be helpful https://github.com/github/feedback/discussions

Edit: I made the suggestion, please give me an upvote on it and comment to say you would like this too.
https://github.com/github/feedback/discussions/2963

**Comment by Zrips** [comment]
Yea, give it a like. I just wonder how long it takes to implement feature like this or would it be faster to do it manually when i have some spare time.

**Comment by Zrips** [comment]
Moved to discussions section.
